code/clement/html_css_flask/lab03/templates/index.html

```diff
@@ -0,0 +1,262 @@
+                                data-bs-toggle="dropdown" style="color: aliceblue;" aria-expanded="false"></a>
+                            <ul class="dropdown-menu" aria-labelledby="navbarDropdown" style="background-color: blue;">
+                                <li><a class="dropdown-item" href="#">About NAS</a></li>
+                                <li><a class="dropdown-item" href="#">Astronauts</a></li>
+                                <li><a class="dropdown-item" href="#">Carriers</a></li>
+                                <li><a class="dropdown-item" href="#">Exhibits & Speakers</a></li>
+                                <li><a class="dropdown-item" href="#">Leaderships</a></li>
+                                <li><a class="dropdown-item" href="#">Locations</a></li>
+                                <li><a class="dropdown-item" href="#">Organizations</a></li>
+                                <li><a class="dropdown-item" href="#">People of NASA</a></li>
+                            </ul>
+                        </li>
+
+
+                        <li>
+                            <a class="nav-link active" style="color: aliceblue;" aria-current="page"
+                                href="#">NASA/Audiences</a>
+                        </li>
+                        <li class="nav-item dropdown">
+                            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button"
+                                data-bs-toggle="dropdown" style="color: aliceblue;" aria-expanded="false"></a>
+                            <ul class="dropdown-menu" aria-labelledby="navbarDropdown" style="background-color: blue;">
+                                <li><a class="dropdown-item" href="#">For Media</a></li>
+                                <li><a class="dropdown-item" href="#">For Educators</a></li>
+                                <li><a class="dropdown-item" href="#">For Syudents</a></li>
+                                <li><a class="dropdown-item" href="#">Exhibits & Speakers</a></li>
+                                <li><a class="dropdown-item" href="#">Grades K-4</a></li>
+                                <li><a class="dropdown-item" href="#">Grades 5-8</a></li>
+                                <li><a class="dropdown-item" href="#">Grades 9-11</a></li>
+                                <li><a class="dropdown-item" href="#">Collages & Universities</a></li>
+                            </ul>
+                        </li>
+                    </ul>
+                    <form class="d-flex">
+                        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+                        <button class="btn btn-outline-success" type="submit">Search</button>
+                    </form>
+                </div>
+            </div>
+        </nav>
+    </div>
+    <!-- ----------Container containing all images------ -->
```

Comment:
Nice job using comments to name your sections. You could also use [semantic elements](https://www.w3schools.com/html/html5_semantic_elements.asp) as well.